javascript-manual/modules/ROOT/partials/quickstart.adoc

```diff
@@ -26,11 +26,10 @@ var neo4j = require('neo4j-driver');
   const URI = '<URI to Neo4j database>'
   const USER = '<Username>'
   const PASSWORD = '<Password>'
-  let driver
 
   try {
-    driver = neo4j.driver(URI,  neo4j.auth.basic(USER, PASSWORD))
-    await driver.verifyConnectivity()
+    const driver = neo4j.driver(URI,  neo4j.auth.basic(USER, PASSWORD))
```

Comment:
This should stay the way it is, otherwise `driver` won't be available out of the scope of the `try/catch`. If we changed it, when you tried to use this example to also run queries you'd find you also have to edit this bit to make it as it currently is.

Reply:
I don't see the variable being used in the try catch in any of the examples provided in the quickstart.

Reply:
It's used in all subsequent examples: `driver.executeQuery()`, `driver.session()`, `driver.close()`. If `driver` is defined inside the `try/catch` of connection, all subsequent examples don't work out of the box.

Reply:
Got it. Fixed.